src/Microsoft.Health.Fhir.SqlServer/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="InvalidContinuationToken" xml:space="preserve">
     <value>The provided continuation token is not valid.</value>
   </data>
+  <data name="NullRowVersion" xml:space="preserve">
+    <value>The row version should never be null.</value>
```

Comment:
Is there more meaning we can give when we need to debug this? Maybe also more context like `The update batch operation failed as no row version was returned`

Reply:
It's something that shouldn't even be possible. So I'm not sure the error message even needs to be in a resource file.

Reply:
Removed it as a resource and updated the error message in e937c26.